pennylane/qnodes/jacobian.py

```diff
@@ -323,7 +327,11 @@ def inds_using(m):
                 if variances_required:
                     grad[:, i] = self._pd_analytic_var(k, flat_args, kwargs, **options)
                 else:
-                    grad[:, i] = self._pd_analytic(k, flat_args, kwargs, **options)
+                    if state_required:
+                        factor = 2 * np.sin(self.h) / (4 * np.sin(self.h / 2))
+                    else:
+                        factor = 1
+                    grad[:, i] = factor * self._pd_analytic(k, flat_args, kwargs, **options)
```

Comment:
Perhaps, to make it a bit cleaner?
```suggestion
                    grad[:, i] = self._pd_analytic(k, flat_args, kwargs, **options)

                    if state_required:
                        grad[:, i] *= 2 * np.sin(self.h) / (4 * np.sin(self.h / 2))
```

Reply:
Just realised that `2 * np.sin(self.h) / (4 * np.sin(self.h / 2))` simplifies down to just `np.cos(self.h / 2)` 😆

Reply:
Resolving - something we can come back to when we add support for differentiating the state with the parameter shift rule.